_security/configuration/enabling-security.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: default
+title: Enabling security
+parent: Configuration
+nav_order: 45
+---
+
+# Enabling security
+
+The default version of OpenSearch comes with Security features pre-installed, however if the Security plugin was [disabled]({{site.url}}{{site.baseurl}}/security/configuration/disable/) or OpenSearch was installed without security, for example, using the minimal distribution method, you can enable the plugin as follows.
+
+A full cluster restart is necessary to enable security features.
+{: .label .label-red }
+
+## Installation steps for prod cluster
+
```

Comment:
Fix formatting on code samples: <img width="874" alt="image" src="https://github.com/opensearch-project/documentation-website/assets/104463754/87392d2c-c060-47ff-862c-0287e0362548">

Reply:
I've replaced these with normal copy.html tags, instead of copy-curl.html and updated the tabs